### ISSUE TITLE
Can Cypress axe find missing aria keywords in the markup?

### ISSUE THREAD

**Issue by senthil-kk20** [component-driven/cypress-axe#109]
Can Cypress axe find missing aria keywords in the markup? I am using it for color contrast checking so far wanted to try for aria keywords too. Is it working for this scenario as well apart from all the things it is doing?
